fundamentals/fundamentals-bio/src/test/scala/izumi/fundamentals/bio/test/BIOSyntaxTest.scala

```diff
@@ -1,12 +1,62 @@
 package izumi.fundamentals.bio.test
 
-import izumi.functional.bio.{BIO, BIOArrow, BIOAsk, BIOBifunctor3, BIOFork, BIOFork3, BIOFunctor, BIOLocal, BIOMonad, BIOMonad3, BIOMonadAsk, BIOMonadError, BIOPrimitives, BIOPrimitives3, BIOProfunctor, BIOTemporal, BIOTemporal3, F}
+import izumi.functional.bio.{BIO, BIOArrow, BIOAsk, BIOBifunctor3, BIOFork, BIOFork3, BIOFunctor, BIOLocal, BIOMonad, BIOMonad3, BIOMonadAsk, BIOMonadError, BIOParallel, BIOPrimitives, BIOPrimitives3, BIOProfunctor, BIOTemporal, BIOTemporal3, F}
 import org.scalatest.wordspec.AnyWordSpec
 
 import scala.concurrent.duration._
 
 class BIOSyntaxTest extends AnyWordSpec {
 
+  "BIOParallel.zipPar/zipParLeft/zipParRight/zipWithPar is callable" in {
+    def x[F[+_, +_]: BIOParallel](a: F[Nothing, Unit], b: F[Nothing, Unit]) = {
+      a zipPar b
+      a zipParLeft b
+      a zipParRight b
+      a.zipWithPar(b)((a, b) => (a, b))
+    }
+
+    x[zio.IO](zio.UIO.succeed(()), zio.UIO.succeed(()))
+  }
+
+  "BIOParallel3.zipPar/zipParLeft/zipParRight/zipWithPar is callable" in {
+    import izumi.functional.bio.BIOParallel3
+    def x[F[-_, +_, +_]: BIOParallel3](a: F[Any, Nothing, Unit], b: F[Any, Nothing, Unit]) = {
+      a zipPar b
+      a zipParLeft b
+      a zipParRight b
+      a.zipWithPar(b)((a, b) => (a, b))
+    }
+
+    x[zio.ZIO](zio.UIO.succeed(()), zio.UIO.succeed(()))
+  }
+
+  "BIOAsync.race is callable along with all BIOParallel syntax" in {
+    import izumi.functional.bio.BIOAsync
+    def x[F[+_, +_]: BIOAsync](a: F[Nothing, Unit], b: F[Nothing, Unit]) = {
+      a zipPar b
+      a zipParLeft b
+      a zipParRight b
+      a.zipWithPar(b)((a, b) => (a, b))
+      a race b
+    }
+
+    x[zio.IO](zio.UIO.succeed(()), zio.UIO.succeed(()))
+  }
+
+  "BIOAsync3.race is callable" in {
+    import izumi.functional.bio.BIOAsync3
+
+    def x[F[-_, +_, +_]: BIOAsync3](a: F[Any, Nothing, Unit], b: F[Any, Nothing, Unit]) = {
+      a zipPar b
```

Comment:
@Adriani277 This keeps working because of the `BIOParallel` import above, with it cleaned up it fails.

<img width="326" alt="Screenshot 2020-05-06 at 19 00 02" src="https://user-images.githubusercontent.com/450507/81211975-15333800-8fcc-11ea-8fd2-b8aa7a12de52.png">
<img width="343" alt="Screenshot 2020-05-06 at 19 01 03" src="https://user-images.githubusercontent.com/450507/81211982-182e2880-8fcc-11ea-856d-fffc2aedd911.png">

Reply:
Ah apologies, seems I missed that BIOParallel import